Utils/release_notes_generator.py

```diff
@@ -119,7 +120,10 @@ def construct_entities_block(entities_data: dict):
             if entity_type in ('Connections', 'IncidentTypes', 'IndicatorTypes', 'Layouts', 'IncidentFields'):
                 release_notes += f'- **{name}**\n'
             else:
-                release_notes += f'##### {name}  \n{description}\n'
+                if add_whitespaces:
+                    release_notes += f'##### {name}  \n{description}\n'
+                else:
+                    release_notes += f'##### {name}\n{description}\n'
```

Comment:
why is this needed?

Reply:
according to @jochman & @yaakovi these spaces are for the `md` parsing in the docs site. for the changelog file in the index, we don't need them, so this boolean variable helps us getting both